fm_api/api.go

```diff
@@ -65,14 +66,16 @@ func (m *FmApiManager) GetQueue() ([]QueueItem, error) {
 func (m *FmApiManager) Listen(c chan bool, l int, r func() string) {
 	for {
 		<-c
-		queue, err := m.GetQueue()
-		if err != nil {
-			log.Printf("Error when fetching queue", err)
-			continue
-		}
+		if time.Now().Hour() <= 18 { // dont run after 6pm
```

Comment:
It plays only when a track finishes.. So it shouldn't play in weekends and bank holidays if someone don't add track into the queue.